internal/utils/misc.go

```diff
@@ -103,6 +103,7 @@ var (
 		"pgbouncer",
 		"pgsodium",
 		"pgsodium_masks",
+		"_realtime",
```

Comment:
This is a workaround for Realtime's DB that keeps track of all tenants and their database credentials.